api/v1/models/Location.js

```diff
@@ -5,15 +5,13 @@ var Location = Model.extend({
     idAttribute: 'id',
     validations: {
         name: ['required', 'string', 'maxLength:255'],
-        shortName: ['required', 'string', 'maxLength:25'],
         latitude: ['required', 'number'],
         longitude: ['required', 'number']
     }
 });
 
 Location.findByName = function (name) {
```

Comment:
I'm honestly not even sure if a find-by-name is useful at all if it requires providing the exact name with proper capitalization and spacing. Do we use it anywhere?